tests/__init__.py

```diff
@@ -27,8 +27,8 @@ def create_authorization_header(service_id=None, key_type=KEY_TYPE_NORMAL):
             secret = api_key.secret
 
     else:
-        client_id = current_app.config['ADMIN_CLIENT_USER_NAME']
-        secret = current_app.config['API_INTERNAL_SECRETS'][0]
+        client_id = current_app.config['ADMIN_CLIENT_ID']
```

Comment:
I agree the behaviour is surprising and there's no need to have it in one method. I'm tempted to refactor it in a follow-up PR as I now realise it could be used to DRY-up [the two request tests](https://github.com/alphagov/notifications-api/pull/3300/files#diff-a705d52d5175ca6300ff24be4c10207e0c197282c3f89d39435b8fb1e46506a3R73).

I'll leave this unresolved and see if it's a quick change.

Reply:
Here's a follow-up PR to address this one: https://github.com/alphagov/notifications-api/pull/3303